pkg/controllers/nodeclaim/tagging/suite_test.go

```diff
@@ -220,9 +221,12 @@ var _ = Describe("TaggingController", func() {
 				Expect(instanceTags).To(HaveKeyWithValue(tag, value))
 			}
 		},
-		Entry("with only karpenter.k8s.aws/nodeclaim tag", v1.TagName),
-		Entry("with only Name tag", v1.TagNodeClaim),
-		Entry("with both Name and karpenter.k8s.aws/nodeclaim tags"),
-		Entry("with nothing to tag", v1.TagName, v1.TagNodeClaim),
+		Entry("with the karpenter.sh/nodeclaim tag", v1.TagName, v1.EKSClusterNameTagKey),
```

Comment:
is this an expected change? from karpenter.k8s.aws to karpenter.sh?

Reply:
Yes, this was wrong originally -- just updating to make it correct